gpii/node_modules/flowManager/src/CloudBasedFlowManager.js

```diff
@@ -40,9 +40,13 @@ fluid.defaults("gpii.flowManager.cloudBased", {
     }
 });
 
-gpii.flowManager.cloudBased.matchToSettings = function (finalPayload, event) {
-    var lifecycleInstructions =
-        fluid.get(finalPayload, [ "matchMakerOutput", "inferredConfiguration", finalPayload.activeContextName, "lifecycleInstructions"]);
+gpii.flowManager.cloudBased.matchToSettings = function (flowManager, finalPayload, event) {
+    var inferredConfiguration = fluid.get(finalPayload, [ "matchMakerOutput", "inferredConfiguration", "gpii-default"]);
```

Comment:
probably a bit neater is var inferredConfiguration = fluid.get( ... ) || {}; if that works